tests/run_standalone_tests.sh

```diff
@@ -41,11 +41,12 @@ parametrizations_arr=($parametrizations)
 report=''
 
 rm -f standalone_test_output.txt  # in case it exists, remove it
+rm -f testnames.txt
```

Comment:
```suggestion
rm -f testnames.txt

```
Maybe this could use an empty line, just for the eyes